src/bundler/bundle_v2.zig

```diff
@@ -2116,8 +2195,8 @@ pub const BundleV2 = struct {
 
                         // Discover all CSS roots.
                         for (import_records.slice()) |*record| {
-                            if (record.tag != .css) continue;
                             if (!record.source_index.isValid()) continue;
+                            if (loaders[record.source_index.get()] != .css) continue;
```

Comment:
This is an extra pointer lookup, which slightly slows things down. Was the `import_record.tag` inaccurate?